src/components/molecules/CheckboxField/CheckboxField.tsx

```diff
@@ -82,15 +82,16 @@ const Label = styled(InputLabel)`
   }
 `;
 
-const CheckboxField: React.FC<IField> = props => {
-  const { label, inputProps, errorMessage, size, ...rest } = props;
-  const { name } = inputProps;
+export interface ICheckboxFieldProps extends IField, IInput {}
 
-  if (!name) throw Error('Name must be set in inputProps. Check the docs.');
+const CheckboxField: React.FC<ICheckboxFieldProps> = props => {
+  const { label, name, errorMessage, inputSize, className, ...rest } = props;
+
+  if (!name) throw Error('Name must be set. Check the docs.');
 
   return (
-    <SizedContainer size={size} {...rest}>
-      <Input id={`checkbox-id-${name}`} type="checkbox" {...inputProps} />
+    <SizedContainer size={inputSize} className={className}>
+      <Input id={`checkbox-id-${name}`} type="checkbox" {...rest} />
```

Comment:
We're then are not adding `name` anymore to this `<Input />`( _[see snapshot diff](https://github.com/zopaUK/react-components/pull/231/files#diff-7682ae740e2ddaa1075bfe3d7b94806f)_ ) is it expected?

Reply:
fixed